t/perf/opcount.t

```diff
@@ -698,6 +698,21 @@ test_opcount(0, "multiconcat: local assign",
 
 # builtin:: function calls should be replaced with efficient op implementations
 no warnings 'experimental::builtin';
+use builtin qw(
+    blessed
+    ceil
+    false
+    floor
```

Comment:
Most of the changes in this file seem like they belong in a separate commit - they have nothing to do with fixing the builtin::indexed optimization

Reply:
The builtin::indexed optimization works fine on `builtin::indexed(...)`, but may crash on `use builtin qw(indexed); ... indexed(...)` because the call optrees look different. I wanted to make sure both cases are optimized correctly (for all builtins).

This change is now a separate commit.